hq/app/config/Config.scala

```diff
@@ -20,16 +22,20 @@ object Config {
     }
   }
 
-  def googleSettings(implicit config: Configuration): GoogleAuthConfig = {
+  def googleSettings(httpConfiguration: HttpConfiguration, config: Configuration): GoogleAuthConfig = {
     val clientId = requiredString(config, "auth.google.clientId")
     val clientSecret = requiredString(config, "auth.google.clientSecret")
     val domain = requiredString(config, "auth.domain")
     val redirectUrl = s"${requiredString(config, "host")}/oauthCallback"
-    GoogleAuthConfig(
+    val secret: String = httpConfiguration.secret.secret
+    val signatureAlgorithm: SignatureAlgorithm = SignatureAlgorithm.forName(httpConfiguration.session.jwt.signatureAlgorithm)
+    GoogleAuthConfig.withSecret(
```

Comment:
Without creating and using a new `GoogleAuthConfig.withSecret()` method, you have two alternative approaches that both use the existing constructor `GoogleAuthConfig.apply()` method in `play-googleauth`.

The closest to your current code is just to construct an `AntiForgeryChecker`, like this :

```
    GoogleAuthConfig(
      clientId,
      clientSecret,
      redirectUrl,
      domain,
      antiForgeryChecker = AntiForgeryChecker(secret,signatureAlgorithm)
    )
```

...this avoids the need to make a new `GoogleAuthConfig.withSecret()` method, but is also strongly-typed - there are fewer `String` arguments floating around to risk getting accidentally swapped.

Better still is to drop the code manually extracting `secret` & `signatureAlgorithm`, and use the `AntiForgeryChecker.borrowSettingsFromPlay(httpConfiguration)` method:

```
    GoogleAuthConfig(
      clientId,
      clientSecret,
      redirectUrl,
      domain,
      antiForgeryChecker = 
        AntiForgeryChecker.borrowSettingsFromPlay(httpConfiguration)
    )
```